tests/scripts/task_mypy.sh

```diff
@@ -29,5 +29,6 @@ mypy  --check-untyped-defs python/tvm/tir/analysis/
 echo "Checking MyPy Type defs in the transform package."
 mypy  --check-untyped-defs python/tvm/tir/transform/
 
-echo "Checking MyPy Type defs in the tvm.relay.backend.contrib.ethosu package."
-mypy  --check-untyped-defs python/tvm/relay/backend/contrib/ethosu/
+#TODO(@mikepapadim): This is failing atm
```

Comment:
Please create an issue if things like this happens and possibly ping codeowners please, we were not aware that any of the checked in code was not tested until recently.

cc : @mikepapadim @junrushao1994

Reply:
Sure. I thought we did, but it turned out haven't...Please make sure to report in time :-)

Reply:
oh actually we reported in this thread: https://github.com/apache/tvm/pull/9050. definitely should submit it as a separate PR though

Reply:
Yea but that PR was closed saying not needed. Thus, I was under the impression that was never merged